doc/source/whatsnew/v0.24.0.txt

```diff
@@ -159,6 +159,30 @@ This is the same behavior as ``Series.values`` for categorical data. See
 :ref:`whatsnew_0240.api_breaking.interval_values` for more.
 
 
+.. _whatsnew_0240.enhancements.multi_index_repr:
+
+Better repr for MultiIndex
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Printing of ``MultiIndex`` instances now shows tuples of each row and ensures
+that the tuple items are vertically aligned, so it's now much much easier to
+understand the structure of the ``MultiIndex``. (:issue:`13480`):
```

Comment:
I would phrase this a bit less strong (what people find looking better / easier to understand is rather subjective). Eg something like "in an attempt to make the structure of the MultiIndex easier to understand"